app/assets/javascripts/jquery.js

```diff
@@ -0,0 +1,5 @@
+/*! jQuery v1.12.4 | (c) jQuery Foundation | jquery.org/license */
```

Comment:
Why using an outdaded version? Current is 3.6.0

Do we really really need jquery? Because that one more thing to keep up to date and this one is easily forgotten

Reply:
As far as know it's needed to get the JS working with the new hmrcLayout which we need for the back links, but I have updated the version to 3.6.0

Reply:
i have deleted all JS as it wasnt needed anymore